static/docs/command-reference/checkout.md

```diff
@@ -21,34 +21,31 @@ which instance of each data file or directory should be used, with the checksums
 saved in the `outs` field. The `dvc checkout` command updates the workspace data
 to match with the <abbr>cached</abbr> files that correspond to those checksums.
 
-Using an SCM like Git, the DVC-files are kept under version control. At a given
-branch or tag of the SCM repository, the DVC-files will contain checksums for
-the corresponding data files kept in the cache. After an SCM command like
-`git checkout` is run, the DVC-files will change to the state at the specified
-branch or commit or tag. Afterwards, the `dvc checkout` command is required in
-order to synchronize the data files with the currently checked out DVC-files.
+When using Git, the DVC-files can be kept under version control. DVC-files in
+different branches or tags may contain checksums for different data files, saved
+by DVC in the project's cache. The DVC-files in the workspace, or the value of
+their `outs` fields can change when using `git checkout`. The `dvc checkout`
+command is required in this situation, in order to synchronize the data files
+(tracked by DVC) with the checked out DVC-files.
 
-This command must be executed after `git checkout` since Git doesn't track files
```

Comment:
Again, not sure why are we removing this ...

Reply:
It's redundant, we've already said what situations require `dvc checkout` and why in the previous paragraph (in the new version of the p, that is). Please see https://github.com/iterative/dvc.org/pull/864#pullrequestreview-337940215 now.